modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1089,6 +1089,105 @@ definition {
 		AppBuilderAdmin.validateIfSearchItemIsNotFound(itemName = "Apple");
 	}
 
+	@description = "Validate empty list state of Apps listing screen"
+	@priority = "5"
+	test ImplementRulesForFieldSets {
+		property portal.acceptance = "true";
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.setUpTwoFieldSets(
+			fieldSetName = "Field Sets",
+			fieldType = "Text",
+			fieldType2 = "Numeric");
+
+		FormViewBuilder.goToFieldsetsTab();
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Field Sets");
+
+		FormViewBuilder.setName(formViewName = "Fields from Fieldsets");
+
+		FormViewBuilder.switchToRulesTab();
+
+		FormViewBuilder.createFirstRule();
+
+		FormViewBuilder.setRuleName(ruleName = "Rule #1");
+
+		FormRules.viewInactiveSaveButton();
+
+		FormRules.configureConditionOptions(
+			conditionField = "Text",
+			conditionOperator = "Is not empty",
+			conditionRowNumber = "1");
+
+		FormRules.configureActionOptions(
+			action = "Show",
+			actionRowNumber = "1",
+			actionTarget = "Numeric");
+
+		FormViewBuilder.ruleSave();
+
+		FormViewBuilder.save();
+
+		AppBuilderAdmin.switchTab(tabName = "Table Views");
+
+		AppBuilderAdmin.add();
+
+		TableViewBuilder.addColumnByDragAndDrop(fieldType = "Text");
+
+		TableViewBuilder.addColumnByDoubleClick(fieldType = "Numeric");
+
+		TableViewBuilder.validateDisabledField(fieldType = "Text");
+
+		TableViewBuilder.validateDisabledField(fieldType = "Numeric");
+
+		TableViewBuilder.setName(tableViewName = "Table View");
+
+		TableViewBuilder.save();
+
+		AppBuilderAdmin.switchTab(tabName = "Apps");
+
+		AppBuilderAdmin.add();
+
+		AppBuilderAdmin.setName(title = "My First App");
+
+		App.selectFormView(formViewName = "Fields from Fieldsets");
+
+		App.selectTableView(tableViewName = "Table View");
+
+		App.selectTableView(tableViewName = "No Workflow");
```

Comment:
"No Workflow" is selected by default. You can replace this with `Button.clickNext();`